<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>637</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="chooseImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>选择图片</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="SelectDetectorFunction">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>-1</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollArea" name="imageScrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>485</width>
      <height>270</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>480</width>
      <height>270</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>469</width>
       <height>254</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>25</y>
      <width>9999</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollArea" name="resultImagescrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>485</width>
      <height>270</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>469</width>
       <height>254</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="SelectPort">
    <property name="geometry">
     <rect>
      <x>752</x>
      <y>451</y>
      <width>120</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="portDataBrowser">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>29</y>
      <width>381</width>
      <height>421</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="SelectBaudRate">
    <property name="geometry">
     <rect>
      <x>753</x>
      <y>477</y>
      <width>120</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openPort">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>540</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearText">
    <property name="geometry">
     <rect>
      <x>493</x>
      <y>451</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>清空消息</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>487</x>
      <y>-49</y>
      <width>3</width>
      <height>9999</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>458</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>串口:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>485</y>
      <width>50</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>波特率:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sendText">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>332</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendData">
    <property name="geometry">
     <rect>
      <x>822</x>
      <y>0</y>
      <width>49</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>701</x>
      <y>512</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>数据位:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="SelectDataLength">
    <property name="geometry">
     <rect>
      <x>753</x>
      <y>504</y>
      <width>120</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="closePort">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>540</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearPort">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>451</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>清空串口</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="resultTextBrowser">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>480</y>
      <width>201</width>
      <height>88</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="sendDetectorResult">
    <property name="geometry">
     <rect>
      <x>214</x>
      <y>0</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>向串口发送结果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearDetectResult">
    <property name="geometry">
     <rect>
      <x>354</x>
      <y>0</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>清空识别结果</string>
    </property>
   </widget>
   <zorder>SelectPort</zorder>
   <zorder>SelectBaudRate</zorder>
   <zorder>SelectDataLength</zorder>
   <zorder>chooseImage</zorder>
   <zorder>SelectDetectorFunction</zorder>
   <zorder>imageScrollArea</zorder>
   <zorder>line_2</zorder>
   <zorder>resultImagescrollArea</zorder>
   <zorder>portDataBrowser</zorder>
   <zorder>openPort</zorder>
   <zorder>clearText</zorder>
   <zorder>line</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>sendText</zorder>
   <zorder>sendData</zorder>
   <zorder>label_3</zorder>
   <zorder>closePort</zorder>
   <zorder>clearPort</zorder>
   <zorder>resultTextBrowser</zorder>
   <zorder>sendDetectorResult</zorder>
   <zorder>clearDetectResult</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
